x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/manifest.json

```diff
@@ -1,29 +1,29 @@
 {
   "id": "apm_transaction",
   "title": "APM",
-  "description": "Detect anomalies in transactions from your APM services.",
+  "description": "Detect anomalies in transactions from your APM services for metric data.",
   "type": "Transaction data",
   "logoFile": "logo.json",
-  "defaultIndexPattern": "apm-*-transaction",
+  "defaultIndexPattern": "apm-*-metric-*",
```

Comment:
is it possible here to use `apm-*-metric,metrics-apm*`, the latter being used for data streams? (also note the lack of a trailing `-` for the first search target)

Reply:
Edited to `apm-*-metric,metrics-apm*` in https://github.com/elastic/kibana/pull/111957/commits/e7ac87e0788e64bcd2de7bf496fdfa9f10ec345e

Reply:
Yes, we do override `indexPatternName`.